src/Microsoft.Extensions.DiagnosticAdapter/Internal/ProxyTypeEmitter.cs

```diff
@@ -101,8 +101,12 @@ private static bool VerifyProxySupport(ProxyBuilderContext context, Tuple<Type,
             }
 
             ProxyTypeCacheResult cacheResult;
+            var verificationResult = new VerificationResult();
             if (context.Cache.TryGetValue(key, out cacheResult))
             {
+                verificationResult.Constructor = cacheResult.Constructor;
+                verificationResult.Type = cacheResult.Type;
+                context.Visited.Add(key, verificationResult);
```

Comment:
Add a blank line after this, it's bad Feng Shui to put a comment directly between two lines of code.